### ISSUE TITLE
"Remove if media does not have alt text"

### ISSUE THREAD

**Issue by derekvanvliet** [skyfeed-dev/query-engine#7]
i would like to make a feed that requires alt text in images. to accomplish this, i was thinking it would be useful to have an option in the Remove block to remove items if their media contains no alt text

**Comment by Invertex** [comment]
You can do this with the RegEx block.
Set it to just "Image Alt Text", and use this RegEx `^\S+$`
This will then only include posts that don't have just whitespace/null alt-text.
